code/_helpers/global_lists.dm

```diff
@@ -231,3 +231,68 @@ var/global/list/string_slot_flags = list(
 */
 //Hexidecimal numbers
 var/global/list/hexNums = list("0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "A", "B", "C", "D", "E", "F")
+
+/**
+ * VOREStation global lists
+*/
+
+var/global/list/ear_styles_list = list()	// Stores /datum/sprite_accessory/ears indexed by type
+var/global/list/tail_styles_list = list()	// Stores /datum/sprite_accessory/tail indexed by type
+var/global/list/wing_styles_list = list()	// Stores /datum/sprite_accessory/wing indexed by type

```

Comment:
Should these not live on glob? And also you appear to have misplaced some vore edit tags

Reply:
These and the other preference setup lists are not GLOB'd assumedly because they need to exist from before GLOB is actually initialized.